client/my-sites/post-type-list/index.jsx

```diff
@@ -48,6 +49,7 @@ class PostTypeList extends Component {
 		// Props
 		query: PropTypes.object,
 		scrollContainer: PropTypes.object,
+		isVipSite: PropTypes.bool,
```

Comment:
This is connected prop, so it should be moved below, under line 54

Reply:
Good call.